java/src/processing/mode/java/JavaEditor.java

```diff
@@ -36,6 +36,9 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
 
+import java.net.URI;
+import java.net.URISyntaxException;
+
```

Comment:
Are these necessary anymore? If not please remove, and then this is ready to be approved! 🎉